src/server/engine_factory_v2.rs

```
@@ -154,18 +145,23 @@ impl TabletFactory<RocksEngine> for KvEngineFactoryV2 {
     }
 
     #[inline]
-    fn tablet_path(&self, id: u64, suffix: u64) -> PathBuf {
+    fn tablet_path_with_prefix(&self, prefix: &str, id: u64, suffix: u64) -> PathBuf {
         self.inner
             .store_path()
-            .join(format!("tablets/{}_{}", id, suffix))
+            .join(format!("tablets/{}{}_{}", prefix, id, suffix))
     }
 
     #[inline]
     fn mark_tombstone(&self, region_id: u64, suffix: u64) {
         let path = self.tablet_path(region_id, suffix).join(TOMBSTONE_MARK);
-        std::fs::File::create(&path).unwrap();
+        let _ = std::fs::File::create(&path);
```

Comment why we could ignore the error?

---

yes, I have added.